README.md

```diff
@@ -72,6 +73,15 @@ See below for how to configure the environment variables appropriately
 - Awattar__VATMultiplier=1.00 # Optional (default: 1.19), you should not need to set this unless your VAT differs from the default
 ```
 
+### Barry
+
+```yaml
+- TeslaMate__EnergyProvider=Barry
+- Barry__BarryApiKey=XXXXX # Required - API key can be optained through their mobile app
+- Barry__BarryMPID=YYYYY # Required - metering point ID (MPID) can be obtained through their mobile app
```

Comment:
Oops, did just spot this little documentation error after renaming the variables!

```suggestion
- Barry__ApiKey=XXXXX # Required - API key can be optained through their mobile app
- Barry__MPID=YYYYY # Required - metering point ID (MPID) can be obtained through their mobile app
```